### ISSUE TITLE
The number of epochs?

### ISSUE THREAD

**Issue by IssamLaradji** [jiwoon-ahn/psa#6]
Thanks for the code!

For how many epochs did you train your model to achieve the results in the paper? I am wondering what a good practice would be for reporting the results since there isn't really a validation set for early stopping. Thanks!

**Comment by jiwoon-ahn** [comment]
All of our models are trained for 7 epochs. I don't think training more or less than this will make a huge difference though.

**Comment by IssamLaradji** [comment]
Thanks! :)
